bin/reflective_loader/src/lib.rs

```diff
@@ -0,0 +1,30 @@
+#![cfg_attr(not(test), no_std)]
+#![cfg_attr(not(test), no_main)]
```

Comment:
To keep the loader DLL "portable" we can't have any relocations. `no_std` `no_main` ensures that we don't link against the C run time.